packages/autorest.python/autorest/codegen/serializers/operation_groups_serializer.py

```diff
@@ -64,6 +65,14 @@ def serialize(self) -> str:
         template = self.env.get_or_select_template(
             "operation_groups_container.py.jinja2"
         )
+
+        pyright_not_report_unnecessary_ignore = any(
```

Comment:
can we add this as aproperty on an lro operation that we bubble up?

Reply:
also shouldn't we just ignore a isngle line, not an entire file in this case?

Reply:
removed this logic since we are now not using `reportUnnecessaryTypeIgnoreComment`.